CHANGELOG.md

```diff
@@ -6,6 +6,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Added
+
+### Changed
+
+- Changed the dispatcher to close epochs based on block numbers instead of block timestamps.
+- Renamed `CARTESI_EPOCH_DURATION` to `CARTESI_EPOCH_LENGTH`, and set its default value to 5760 (1 day worth of blocks, in average).
```

Comment:
Please use `CARTESI_EPOCH_BLOCK_LENGTH` instead.

```suggestion
- **BREAKING**: replaced `CARTESI_EPOCH_DURATION` with `CARTESI_EPOCH_BLOCK_LENGTH` to match the new epoch calculation procedure, and set its default value to 5760 (1 day worth of blocks, in average).
```

Reply:
I also feel there should be a reference to the `ClaimMismatch` error being fixed. Something like:

```
### Fixed
- Fixed a bug that caused the node to create a ClaimMismatch error when reprocessing inputs after a reboot.
```

Reply:
I also feel there should be a reference to the `ClaimMismatch` bug being fixed. Something like:

```
### Fixed
- Fixed a bug that caused a `ClaimMismatch` error when reprocessing inputs after a reboot.
```